src/Bicep.Core/Analyzers/Linter/Rules/NoUnusedParametersRule.cs

```diff
@@ -2,7 +2,6 @@
 // Licensed under the MIT License.
```

Comment:
Please add this to NoUnusedParametersRuleTests.cs and a similar one for NoUnusedVariablesRuleTests.cs:

 [DataRow(@"param")] // Don't show as unused - no param name
        [DataRow(@"param // whoops")] // Don't show as unused - no param name
        [DataTestMethod]
        public void Errors(string text, params string[] unusedParams)
        {
            CompileAndTest(text, OnCompileErrors.Ignore);
        }

Reply:
Added your suggestion.